src/frontends/paddle/src/op/elementwise_ops.cpp

```diff
@@ -46,6 +46,10 @@ NamedOutputs elementwise_greater_equal(const NodeContext& node_context) {
     return elementwise_ops<default_opset::GreaterEqual>(node_context);
 }
 
+NamedOutputs elementwise_not_equal(const NodeContext& node_context) {
+    return elementwise_ops<default_opset::NotEqual>(node_context);
```

Comment:
“NotEqual” Operator can reuse 'elementwise_ops' base process, but such kind of elementwise related Ops are recommanded to use separate conversions(like, [greater_than](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/paddle/src/op/greater_than.cpp) )